### ISSUE TITLE
Alert notifications stopped working

### ISSUE THREAD

**Issue by sapiderman** [hyperjumptech/monika#805]
**Describe the bug**
Not getting response alert notifications (from alert queries).

**To Reproduce**
1. Using the following config `monika.yaml`:
```yaml
probes:
  - id: probe-01
    name: 'test'
    interval: 10

    requests:
      - url: http://httpbin.org/status/400
        method: GET
        timeout: 5000
        saveBody: false
        headers:
          Content-Type: application/json; charset=utf-8
    alerts:
      - query: response.status != 200
        message: Http Response status code is not 200!
      - query: response.time > 10000
        message: Yo too slow!
    incidentThreshold: 1
    recoveryThreshold: 1

notifications:
  - id: unique-id-desktop
    type: desktop

```
2. Run monika
3. Detecting http.status not 200 but not gettting alerts

**Expected behavior**
1. Should be getting response alert from http status not 200
2. Staring monika alert works

**Screenshots**
![image](https://user-images.githubusercontent.com/964106/180387037-a1df18e2-f251-47c8-b92b-fad881989b1a.png)

**Desktop (please complete the following information):**
- OS: Linux
- Release: 22.04
- Distro: Pop!_OS 22.04 LTS jammy
- Monka Version: 1.9.1

**Additional context**
- SMTP chanell has similar issue, no alert notification
- SMTP channel also receive Monika Start notification 
- Threshold > 1 works

**Comment by sapiderman** [comment]
oh hold on, let me try that

**Comment by sapiderman** [comment]
> I could reproduce this, but seems like it happened because the incidentThreshold is 1. When I set to 2, I got notification.

Just confirmed this. So the issue is with threshold = 1 it seems. Pretty wierd.

**Comment by ilmiawan** [comment]
this is caused by @kevin-hyperjump fixes on issue #578 
![Screen Shot 2022-08-11 at 10 33 09](https://user-images.githubusercontent.com/8952493/184061157-fadd59f0-4f87-4d71-84fc-3dcb3f006293.png)

https://github.com/hyperjumptech/monika/pull/620/files, the fixes on these files.
